galaxy/common_cleanup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -x
+
+# This usually drastically reduced the container size
+# at the cost of the startup time of your application
+find / -name '*.pyc' -delete
+
+find / -name '*.log' -delete
+find / -name '.cache' -delete
+find / -name '.npm' -delete
```

Comment:
I spotted a few folders called `.parcel-cache` under galaxy/config/plugins with significant size, maybe that can be removed as well

Reply:
/root/.launchpadlib is 2MB and added in the "apt-get -qq update     && apt install -y software-properties-common dirmngr gpg gpg-agent   " step

Reply:
will remove these...thanks for noticing!